.github/workflows/release.yml

```diff
@@ -36,14 +39,17 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@9bb56186c3b09b4f86b1c65136769dd318469633
+        with:
+          ref: ${{ inputs.use_existing_tag == 'true' && inputs.version_number || github.ref }}
       - name: Unshallow
         run: git fetch --prune --unshallow
       - name: Create release tag
-        uses: rickstaa/action-create-tag@a1c7777fcb2fee4f19b0f283ba888afa11678b72 # will fail if existing tag is present
+        uses: rickstaa/action-create-tag@a1c7777fcb2fee4f19b0f283ba888afa11678b72
         with:
           tag: ${{ inputs.version_number }}
           gpg_private_key: ${{ secrets.GPG_PRIVATE_KEY }}
           gpg_passphrase: ${{ secrets.PASSPHRASE }}
+          tag_exists_error: ${{ inputs.use_existing_tag == 'true' && false || true }}
```

Comment:
Ref: https://github.com/rickstaa/action-create-tag?tab=readme-ov-file#tag_exists_error
This attribute, if set to true fails the workflow with an error if the tag is already existing, if we want to use an existing tag (`use_existing_tag`), we're essentially bypassing this

Reply:
could it be simplified? like:
tag_exists_error: ${{ inputs.use_existing_tag != 'true' }}

Reply:
updated